chrome/updater/updater_module.cc

```diff
@@ -276,6 +276,7 @@ void UpdaterModule::Update() {
             component.name = "cobalt";
             component.app_id = ids[0];
             component.version = manifest_version;
+            component.sb_version = std::to_string(SB_API_VERSION);
```

Comment:
Is this component.sb_version used anywhere? Looks like the code uses `std::to_string(SB_API_VERSION` when setting the sbversion in the prefs.